includes/subscriptions/templates/html-subscriptions-plugin-notice.php

```diff
@@ -20,12 +20,13 @@
 					<p>
 						<?php
 							printf(
-								// Translators: placeholders are opening and closing strong HTML tags.
-								esc_html__( 'By deactivating the %1$sWooCommerce Subscriptions%2$s plugin, your store will switch to using the subscriptions functionality %1$sbuilt into WooCommerce Payments%2$s. %1$s%3$sLearn more.%4$s%2$s', 'woocommerce-payments' ),
+								// Translators: placeholders are opening and closing strong HTML tags. Last one one adds WooPayments.
+								esc_html__( 'By deactivating the %1$sWooCommerce Subscriptions%2$s plugin, your store will switch to using the subscriptions functionality %1$sbuilt into %5$s%2$s. %1$s%3$sLearn more.%4$s%2$s', 'woocommerce-payments' ),
```

Comment:
Since "WooCommerce Subscriptions is also a brand name, I think we can safely extract it also. You can specify in the translators comment: `%5$s: WooPayments, %6$s: WooCommerce Subscriptions` to be more precise

Reply:
fixed in https://github.com/Automattic/woocommerce-payments/pull/6581/commits/b1d36c896aa92104e381f3d5d35fad7804264515